lib/savolvw.rb

```diff
@@ -28,22 +29,23 @@ def parse
       parsed_email = JSON.parse(@email.body.gsub('!@#', '')) rescue nil
 
       if parsed_email.nil?
-        parsed_email = @email.body.colons_to_hash(/(Telefone|Nome|Mensagem|E-mail|CPF).*?:/, false) unless parsed_email
```

Comment:
I remove the `unless parsed_email` because the `parsed_email` already comes nil.